packages/console/src/components/CustomUiAssetsUploader/index.tsx

```diff
@@ -28,7 +30,19 @@ function CustomUiAssetsUploader({ disabled, value, onChange }: Props) {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const [file, setFile] = useState<File>();
   const [error, setError] = useState<string>();
+  const [abortController, setAbortController] = useState(new AbortController());
   const showUploader = !value?.id && !file && !error;
+  const { setIsUploading, setCancelUpload } = useContext(SignInExperienceContext);
```

Comment:
Since this component relies on the `SignInExperienceContext`, better to place it in the `pages/sign-in-experience` folder to avoid misuse.

Reply:
Updated